### PR TITLE
Update file disk to ^1.0.1

### PR DESCRIPTION
**Pull request by zvin**

### CODE REVIEW

**Review by jhermsmeier — APPROVED** [review]

File: .travis.yml [hunk 1]
```diff
@@ -2,7 +2,7 @@ language: node_js
 before_install:
   - npm install -g npm@latest
 node_js:
-  - 4
```
Comment (jhermsmeier) [inline .travis.yml]: Why remove Node v4? As long as it's LTS, I think we should still test against it.

Comment (zvin) [reply .travis.yml]: We depend on file-disk which uses default parameters in some functions. This doesn't work in node4: http://node.green/#ES2015-syntax-default-function-parameters

Comment (zvin) [reply .travis.yml]: I'll update file-disk to work on node 4 if it is required.

Comment (jhermsmeier) [reply .travis.yml]: Yeah, I think as long as it's LTS we should support it – there are a surprising amount of people is still on Node 4.

**Pull request merged**